tests/foreman/api/test_contentview.py

```diff
@@ -1199,8 +1215,9 @@ def test_custom_cv_subscribe_system(self):
         # This test is implemented in tests/foreman/smoke/test_api_smoke.py.
         # See the end of method TestSmoke.test_smoke.
 
+    @tier2
     @stubbed()
-    def test_cv_dynflow_restart_promote(self):
+    def test_positive_dynflow_restart_promote(self):
```

Comment:
To have same approach as in UI and basically action should be located on third place
`test_positive_restart_promote_via_dynflow`